sk/src/routes/[id]/+page.svelte

```diff
@@ -15,7 +15,7 @@
   $: entry = data.entry
   $: media = data.media
 
-  $: groupped = Object.groupBy(entry.expand?.trs || [], ({ releaseGroup }) => releaseGroup) as Record<string, TorrentsResponse[]>
+  $: groupped = Object.groupBy(sortTorrents(entry.expand?.trs), ({ releaseGroup }) => releaseGroup) as Record<string, TorrentsResponse[]>
```

Comment:
cant trs be null?

Reply:
First line of sortTorrents does a !torrents return []. So check was redundant.